15_totalIntegers/solution/totalIntegers-solution.js

```diff
@@ -0,0 +1,21 @@
+const totalIntegers = function (obj, count = 0) {
```

Comment:
With `count` not actually being passed to any recursive calls, I think it's better to just define it in the function body with a `let count = 0` somewhere.